test/libsolidity/syntaxTests/controlFlow/storageReturn/modifier_err.sol

```diff
@@ -18,5 +18,4 @@ contract C {
     }
 }
 // ----
-// TypeError 3464: (249-258): This variable is of storage pointer type and can be returned without prior assignment, which would lead to undefined behaviour.
-// TypeError 3464: (367-376): This variable is of storage pointer type and can be returned without prior assignment, which would lead to undefined behaviour.
```

Comment:
``f(true)`` *does* return an uninitialized storage pointer and this error is vital - so there is something off in the analysis here.